go/vt/vtctl/reparentutil/emergency_reparenter.go

```diff
@@ -497,6 +496,8 @@ func (erp *EmergencyReparenter) reparentReplicas(
 	)
 
 	replCtx, replCancel := context.WithTimeout(context.Background(), opts.WaitReplicasTimeout)
+	primaryCtx, primaryCancel := context.WithTimeout(context.Background(), topo.RemoteOperationTimeout)
```

Comment:
From looking at this function, I can see that we're not using the context that was passed into the function as the parent context here. We should, shouldn't we?

Reply:
Yes good point, I'll fix it.